vectordatasource/transform.py

```diff
@@ -8906,6 +8927,132 @@ def remap_viewpoint_kinds(shape, props, fid, zoom):
     return (shape, props, fid)
 
 
+def _list_of_countries(value):
+    """
+    Parses a comma or semicolon delimited list of ISO 3166-1 alpha-2 codes,
+    discarding those which don't match our expected format.
+
+    Returns a list of lower-case, stripped country codes.
+    """
+
+    from re import match
+    from re import split
+
+    countries = []
+    candidates = split('[,;]', value)
+
+    for candidate in candidates:
+        # should have an ISO 3166-1 alpha-2 code, so should be 2 ASCII
+        # latin characters.
+        candidate = candidate.strip().lower()
+        if match('[a-z][a-z]', candidate):
```

Comment:
Natural Earth has an `iso` "country code", which is used for disputed boundaries too. Special case that?

Reply:
Sure. Added in 43fe2474.